apps/authorization/management/commands/update_access_grants.py

```diff
@@ -1,27 +1,35 @@
 from django.core.management.base import BaseCommand
 from apps.authorization.models import DataAccessGrant
 from apps.dot_ext.models import Application
+
+
 class Command(BaseCommand):
     help = (
         'Update Access Grants Per Application.'
         'Pass in a list of application ids.'
     )
 
     def add_arguments(self, parser):
-        parser.add_argument('--applications', help="list of applications to update "
+        parser.add_argument('--applications', help="List of applications to update "
                                                     "id, comma separated: "
-                                                    "eg. 1,2,3 ")
+                                                    "eg. 1,2,3. Supersedes --all")
+        parser.add_argument('--all', action='store_true', help="Update access grants for all applications")
+        parser.set_defaults(all=False)
 
 
     def handle(self, *args, **options):
         if options['applications']:
             application_ids = options['applications'].split(',')
+        elif options['all']:
+            application_ids = Application.objects.values_list('id', flat=True)
```

Comment:
I'm pretty sure the django orm compiles very similar sql with this pattern to if i iterated over the queryset directly below. I think this way is more readable, but may shuffle the whole thing to make the options mutually exclusive.